### PR TITLE
Dark Mode styles for news sidebar

### PR DESCRIPTION
**Pull request by Uri-E**
Added dark mode styles to news sidebar. Also made a small correction to timeline spacing, since I'm already here >.>

### CODE REVIEW

**Comment by prd409-utsa** [comment]
Looks good except the border-bottom is showing after the last link on Related News.  I had used a style for :last-child in the regular/light mode to prevent that from happening. If this is being too picky for the DLS version then just disregard....but thought I would point it out. 

Otherwise, looks good!

Screenshot 2024-11-12 at 3 03 33 PM

**Review by prd409-utsa — COMMENTED** [review]
Looks good except the border-bottom is showing after the last link on Related News. I had used a style for :last-child in the regular/light mode to prevent that from happening. If this is being too picky for the DLS version then just disregard....but thought I would point it out.

Otherwise, looks good!

Screenshot 2024-11-12 at 3 03 33 PM

**Review by lalov1 — COMMENTED** [review]
Just the item Pamela mentioned, but everything else looks good! 👍🏽

File: components/03-sections/02-navigation/04-side-navigation-news/side-navigation-news.scss [hunk 1]
```diff
@@ -89,4 +90,62 @@
...
+		.sidebar-news-container {
+			background-color: #151515;
+			.sidebar-news-title {
+				color:$white;
+			}
+			ul.related-news-items {
+				li {
+					border-bottom: 1px solid $grey-b;
+					color:$white;
+					a {
```
Comment (garza-utsa) [inline components/03-sections/02-navigation/04-side-navigation-news/side-navigation-news.scss:107]: see the :last-child stuff missing from line 27 in this same file

**Review by garza-utsa — CHANGES_REQUESTED** [review]
see @prd409-utsa 's comments on border styles :last-child

**Review by prd409-utsa — APPROVED** [review]
Bottom border after third item is no longer showing. Looks good!

**Review by lalov1 — APPROVED** [review]
👍🏽

**Pull request merged**